app/views/helpers/ResultsPageHelper.scala

```diff
@@ -382,7 +384,8 @@ object ResultsPageHelper extends ViewHelper {
               TableRow(content = Text(PercentageUtils.format(s.details.taxRate)))
             )
           ).filter(_.nonEmpty),
-          caption = None,
+          caption = Some("Effective tax rate breakdown for your accounting period"),
```

Comment:
Do we need translations for this?

Reply:
Yeah, i had created the string var but missed it in a couple of places